verification/curator-service/ui/cypress/plugins/index.ts

```diff
@@ -0,0 +1,19 @@
+// Need to use require in plugins file 
+// https://stackoverflow.com/questions/59743577/is-there-anyway-to-query-mongodb-in-cypress-test
+const MongoClient = require('mongodb').MongoClient;
+const url = "mongodb://localhost:27017/";
+
+module.exports = (on: any, config: any) => {
```

Comment:
what's the advantage of doing this as opposed to calling the existing db setup script with `cy.exec('npm run...')`? the db-setup script will also make sure that the right schema is applied, and indexes when we have them (imaging a scenario where someone's testing changes to the validator and it gets out of sync)

(then you also wouldn't have to worry about which db to connect to above)

Reply:
Worked with Kelly offline. The `npm run setup-db` command takes ~7 seconds to run. We don't want to add 7s to each test run, but that is okay for initial testing setup. So I changed the code to run setup-db at the start of the entire test suite, which will init the db with new schema and indexes. Then on each test run it will make this call which runs in <1s to remove the cases.